src/coreclr/jit/llvm.cpp

```diff
@@ -245,12 +316,13 @@ bool Llvm::callHasShadowStackArg(GenTreeCall* call)
 // Return Value:
 //    Reference to the info structure for "helperFunc".
 //
-const HelperFuncInfo& Llvm::getHelperFuncInfo(CorInfoHelpFunc helperFunc)
+/* static */ const HelperFuncInfo& Llvm::getHelperFuncInfo(CorInfoHelpFunc helperFunc)
 {
     // Note on Runtime[Type|Method|Field]Handle: it should faithfully be represented as CORINFO_TYPE_VALUECLASS.
     // However, that is currently both not necessary due to the unwrapping performed for LLVM types and not what
     // the Jit expects. When deleting the unwrapping, fix the runtime signatures to take the underlying pointer instead.
-    const int CORINFO_TYPE_RT_HANDLE = CORINFO_TYPE_PTR;
+    const int CORINFO_TYPE_RT_HANDLE = CORINFO_TYPE_NATIVEINT;
```

Comment:
Due to https://github.com/dotnet/runtimelab/issues/2151, runtime handles show up in signatures as `int32`s.